model/migrations/88cf753-conversation_state.sql

```diff
@@ -0,0 +1,14 @@
+--
+-- Table structure for table `conversation_state`
+--
+DROP TABLE IF EXISTS `conversation_state`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `conversation_state` {
+  `conversationId` varchar(255) COLLATE utf8mb4_bin not NULL,
+  `history` text COLLATE utf8_bin NULL,
```

Comment:
We should probably make the text fields `utf8mb4_bin` because they can contain non-BMP characters (emojis) if the user types them.